lib/jekyll/commands/serve.rb

```diff
@@ -219,17 +223,11 @@ def webrick_opts(opts)
         #
 
         private
-        def start_up_webrick(opts, destination)
-          if opts["livereload"]
-            @reload_reactor.start(opts)
-          end
-
+        def setup_webrick(opts, destination)
           @server = WEBrick::HTTPServer.new(webrick_opts(opts)).tap { |o| o.unmount("") }
```

Comment:
I think that it may be possible to erase the instance variable `@server` by using `attr_accessor` .